http_protocol_proposal_v1.md

```diff
@@ -62,7 +62,7 @@ Implementation MUST provide read access to `Correlation-Context` for logging sys
 
```

Comment:
when you say `Implementation MUST provide read access to `Correlation-Context` for logging systems and MUST support adding properties to Correlation-Context.` that feels a little confusing, because throughout the document you say Correlation-Context is read-only, which implies properties cannot be added to the collection. I think what you really mean is existing properties in the collection should be treated as read only but new properties are allowed.